builder/lxc/config.go

```diff
@@ -48,6 +61,10 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 		c.OutputDir = fmt.Sprintf("output-%s", c.PackerBuildName)
 	}
 
+	if c.ExportName == "" {
+		c.ExportName = "rootfs.tar.gz"
```

Comment:
I like how you're defaulting here to the old behaviour. But maybe we can push it all the way. The default case of exporting the entire rootfs is equivalent to something like this, right?:

```
"export_name": "rootfs.tar.gz",
"export folders": [
  {"src": "/", "dest": "/rootfs" }
]
```

If we can reduce the default case like this, then maybe we can avoid an unnecessary if/else here: https://github.com/saucelabs/packer-builder-lxc/blob/cb64d5ba15d74c3d12cac6d9836ae81157bd293e/builder/lxc/step_export.go#L88-L92